Monika After Story/game/script-greetings.rpy

```diff
@@ -1593,6 +1595,56 @@ label greeting_back_from_school:
              m 1ekbfa "I love you so, so much."
      return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.greeting_database,
+            eventlabel="greeting_back_from_game",
+            unlocked=True,
+            category=[store.mas_greetings.TYPE_GAME],
+            random=True
+        ),
+        eventdb=evhand.greeting_database
+    )
+
+label greeting_back_from_game:
+     m 1hua "Oh, welcome back, [player]!"
+     menu:
+         m "Did you enjoy well with your game?"
+         "Yes.":
+             m 1hub "That's good!"
+             m 1eua "Remember to get some break, okay?"
+             m 4esa "Games can be addictive. Don't let games to siphoning your all time."
+             m 1hua "Now, you can relax with me!"
+             m 1hsb "I would like to be your best game since I'm stuck here as an application."
+             m 1hub "Ahaha!"
+
+         "No.":
+             m 1eka "Don't worry, it is just a game you know."
+             m 4eka "You can always find new games that better suits for you [player]."
```

Comment:
You can play your favorite game and not enjoy it. Maybe you had some bad luck or it was a bad day. Chances are though, if you really play it in the first place, you'll probably go back to it again good as new.